Removes an unnecessary block from meson.build that checks for python
modules and would force python3-* packages into makedepends instead of depends.

diff --git a/meson.build b/meson.build
index b47c7ed..3d91713 100644
--- a/meson.build
+++ b/meson.build
@@ -10,14 +10,6 @@ pkgdatadir = join_paths(prefix, get_option('datadir'), meson.project_name())
 py_installation = pymod.find_installation(get_option('python'))
 py_path = py_installation.get_install_dir()
 
-python3_required_modules = ['gi', 'pyudev', 'xdg', 'evdev', 'gettext', 'matplotlib', 'scipy', 'numpy']
-foreach p : python3_required_modules
-  script = 'import importlib.util; import sys; exit(1) if importlib.util.find_spec(\''+ p +'\') is None else exit(0)'
-  if run_command(py_installation, '-c', script, check: true).returncode() != 0
-    error('Required Python3 module \'' + p + '\' not found')
-  endif
-endforeach
-
 udev_rules_dir = get_option('udev_rules_dir')
 if udev_rules_dir == ''
   if prefix == '/usr/local'
